### PR TITLE
Add separate infrastructure metric set

### PR DESCRIPTION
**Pull request by yngveaasheim**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by yngveaasheim** [comment]
@olaaun Please review this carefully

File: metrics/src/main/java/ai/vespa/metrics/ControllerMetrics.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+    // Metrics per API, metrics created in ControllerMaintainer/MetricsReporter
+    OPERATION_API("operation.api.last", Unit.REQUEST, "Controller: Requests for /api API"),
+    OPERATION_APPLICATION("operation.application", Unit.REQUEST, "Controller: Requests for /application API"),
+    OPERATION_CONFIGSERVER("operation.configserver", Unit.REQUEST, "Controller: Requests for /configserver API"),
+    OPERATION_CONTROLLER("operation.controller", Unit.REQUEST, "Controller: Requests for /controller API"),
+    OPERATION_FLAGS("operation.flags", Unit.REQUEST, "Controller: Requests for /flags API"),
+    OPERATION_OS("operation.os", Unit.REQUEST, "Controller: Requests for /os API"),
+    OPERATION_ROUTING("operation.routing", Unit.REQUEST, "Controller: Requests for /routing API"),
+    OPERATION_ZONE("operation.zone", Unit.REQUEST, "Controller: Requests for /zone API"),
```
Comment (olaaun) [inline metrics/src/main/java/ai/vespa/metrics/ControllerMetrics.java:49]: There's also a `operation.changemanagement`, but it was perhaps intentionally left out?

**Comment by olaaun** [comment]
Include [athenz.request.error](https://github.com/vespa-engine/vespa/blob/master/controller-server/src/main/java/com/yahoo/vespa/hosted/controller/athenz/impl/AthenzClientFactoryImpl.java#L25) in ControllerMetrics?

File: metrics/src/main/java/ai/vespa/metrics/ControllerMetrics.java (hunk 1 above)
Comment (yngveaasheim) [reply metrics/src/main/java/ai/vespa/metrics/ControllerMetrics.java:49]: Not intentional. There was a discrepancy in what what used in dashboards and what was actually sent to Yamas.

Comment (yngveaasheim) [reply metrics/src/main/java/ai/vespa/metrics/ControllerMetrics.java]: Fixed in later commit.

**Review by olaaun — APPROVED** [review]

**Comment by aressem** [comment]
This broke some internal system tests

**Comment by olaaun** [comment]
> This broke some internal system tests

Fixed test

**Pull request merged**